File: practice/7_searching/1_linear.js
// Pseudocode
// ->This function accepts an array and a value
// ->Loop through the array and check if the current array element is equal to the value
// ->If it is equal, return the index at which the element is found
// ->If the value is never found, return -1
function linear(arr,val){
    let c = 0;
    for(let i=0;i<arr.length;i++){
        if(arr[i]==val){
            console.log(i);
            c++;
        }
    }
    if(!c){
        console.log("-1");
    }
}

linear([1,2,3],3)

File: practice/7_searching/1_linear.py
# Pseudocode
# ->This function accepts an array and a value
# ->Loop through the array and check if the current array element is equal to the value
# ->If it is equal, return the index at which the element is found
# ->If the value is never found, return -1
def linear(arr,val):
    c = 1
    for i in range(0,len(arr)):
        if arr[i] == val:
            c -= 1
            print(arr.index(val))
    if c:
        print(-1)        

linear([1,2,3],5)